src/components/settings/SpendingLimits/NewSpendingLimit/steps/SpendingLimitForm.tsx

```diff
@@ -36,16 +37,6 @@ type Props = {
   onSubmit: (data: NewSpendingLimitData) => void
 }
 
-export const _validateSpendingLimit = (val: string, decimals?: number) => {
-  // Allowance amount is uint96 https://github.com/safe-global/safe-modules/blob/master/allowances/contracts/AlowanceModule.sol#L52
```

Comment:
There is a contractual upper limit to allowance amounts so we need to keep this as it's possible to have more assets than the limit, i.e. uint96 + 1.